themes/docsy/layouts/partials/section-index.html

```diff
@@ -18,9 +18,9 @@
         {{ range $pages }}
             {{ if eq .Parent $parent }}
                 <div class="entry">
-                    <h5>
+                    <h3>
```

Comment:
This folder is a sub-module(i,e vendored), and therefore should not be changed. If we want to make a change to any of these files, we need to copy them in the root folder and make these changes. 

For example, this particular file needs to be copied to the [partials](https://github.com/operator-framework/olm-docs/tree/master/layouts/partials) folder in our root to make these changes.

Reply:
I have undone the changes to vendor files and created a new file in the layouts/partials directory.